lib/screens/activity_list_view.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:flutter/material.dart';
+import 'package:maui/components/topic_button.dart';
+import 'package:maui/db/entity/activity.dart';
+import 'package:maui/loca.dart';
+import '../repos/activity_repo.dart';
+
+class ActivityListView extends StatefulWidget {
+  const ActivityListView({Key key}) : super(key: key);
+
+  @override
+  _ActivityListViewState createState() {
+    return new _ActivityListViewState();
+  }
+
+  showModes(BuildContext context, String game) {
+    Navigator.of(context).pushNamed('/categories/$game');
+  }
+}
+
+class _ActivityListViewState extends State<ActivityListView> {
+  List<Activity> _dataActivity;
+  bool _isLoading = true;
+  static const activityTopicId = 'lion';
```

Comment:
Take this as an input. For now, if you want to test, you can make it take a hard coded value, like 'lion' in the constructor